### PR TITLE
Add DNS_INFORMATION_DISCLOSURE KB

### PR DESCRIPTION
**Pull request by elyousfi5**
PR to add DNS_INFORMATION_DISCLOSURE KB

### CODE REVIEW

File: WEB_SERVICE/DNS/_HIGH/DNS_INFORMATION_DISCLOSURE/meta.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+  "risk_rating": "high",
```
Comment (nmasdoufi-ol) [inline WEB_SERVICE/DNS/_HIGH/DNS_INFORMATION_DISCLOSURE/meta.json:2]: - Medium Risk: If the disclosure only includes basic public-facing information (e.g., web and mail server records).
- High Risk: If it reveals internal infrastructure, exposes AXFR, or supports further targeted attacks.

Comment (elyousfi5) [reply WEB_SERVICE/DNS/_HIGH/DNS_INFORMATION_DISCLOSURE/meta.json:2]: If you read the description you will see that its about the `API Keys and Passwords` and private ips

File: WEB_SERVICE/DNS/_HIGH/DNS_INFORMATION_DISCLOSURE/meta.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  "short_description": "DNS records expose sensitive information, potentially aiding attackers in gathering intelligence or executing attacks.",
+  "references": {
+    "DNS Information Disclosure": "https://owasp.org/www-community/vulnerabilities/DNS_Zone_Transfer",
```
Comment (PiranhaSa) [inline WEB_SERVICE/DNS/_HIGH/DNS_INFORMATION_DISCLOSURE/meta.json:5]: the link doesn't work and it seems to talk  about dns_zone_transfer

File: WEB_SERVICE/DNS/_HIGH/DNS_INFORMATION_DISCLOSURE/meta.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    "Sensitive Data Exposure": "https://owasp.org/www-project-top-ten/OWASP_Top_Ten_2017/Top_10-2017_A3-Sensitive_Data_Exposure"
+  },
+  "title": "DNS Information Disclosure",
+  "privacy_issue": true,
+  "security_issue": true,
+  "categories": {
+    "SOC2_CONTROLS": [
+      "CC_3_4",
```
Comment (PiranhaSa) [inline WEB_SERVICE/DNS/_HIGH/DNS_INFORMATION_DISCLOSURE/meta.json:13]: don't think it's the case 
```suggestion
```

File: WEB_SERVICE/DNS/_HIGH/DNS_INFORMATION_DISCLOSURE/meta.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      "CC_4_1",
+      "CC_6_1",
+      "CC_6_6",
+      "CC_6_7",
```
Comment (PiranhaSa) [inline WEB_SERVICE/DNS/_HIGH/DNS_INFORMATION_DISCLOSURE/meta.json]:
```suggestion

```

Comment (elyousfi5) [reply WEB_SERVICE/DNS/_HIGH/DNS_INFORMATION_DISCLOSURE/meta.json]: image
image

**Review by 3asm — APPROVED** [review]

**Pull request merged**